### PR TITLE
Added Support For Teltonika RUT2XX Devices

### PR DESCRIPTION
**Pull request by jozefrebjak**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/rutos-2xx.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+modules:
+  sensors:
+    pre-cache:
+      data:
+        - oid:
+            - Operator
+    temperature:
+      data:
+        - oid: Temperature
+          value: Temperature
```
Comment (murrant) [inline includes/definitions/discovery/rutos-2xx.yaml]: You don't need value if oid is the same.

Comment (jozefrebjak) [reply includes/definitions/discovery/rutos-2xx.yaml]: I deleted them.

File: includes/definitions/rutos-2xx.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+os: rutos-2xx
+type: network
+group: unix
+text: "Teltonika RutOS"
+icon: teltonika
+ifXmcbc: true
+ifname: true
+mib_dir:
+  - rutos-2xx
```
Comment (murrant) [inline includes/definitions/rutos-2xx.yaml]: The mib directory should be `teltonika` can you change that and move the mib?

File: includes/polling/os/rutos-2xx.inc.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?php
+$hardware = trim(snmp_get($device, '1.3.6.1.4.1.48690.1.7.0', '-OQv', '', ''), '"');
```
Comment (murrant) [inline includes/polling/os/rutos-2xx.inc.php]: trim() functions are not needed.

File: includes/definitions/rutos-2xx.yaml (hunk 2 above)
Comment (jozefrebjak) [reply includes/definitions/rutos-2xx.yaml]: done, but there is also MIB for Rut9xx, which I am not using for now, but there is a support for that and it's in a rutos folder, and there is also used TELTONIKA-MIB, but that MIB is different for RUT9XX devices, but same name, so what can we do with that ? create another folder under Teltonika for RUT2XX and another folder for RUT9XX and then move MIBs with a same name here ?

File: includes/polling/os/rutos-2xx.inc.php (hunk 3 above)
Comment (jozefrebjak) [reply includes/polling/os/rutos-2xx.inc.php]: I changed it completely to OIDs from MIB, they are didn't worked for me, but I found where was a problem.

File: includes/definitions/rutos-2xx.yaml (hunk 2 above)
Comment (murrant) [reply includes/definitions/rutos-2xx.yaml]: yeah, I think subfolders is the best option there.  Silly folks using the same mib name.
The cambium mibs also deal with this.

**Comment by murrant** [comment]
@jozefrebjak why is this different from rutos?

**Comment by jozefrebjak** [comment]
rutos is designed for RUT9XX devices, this is for 2XX which are using different MIB, but with same name. Maybe we can add 2xx to rutos and update rutos, where are missing some sensors, but how to use two different MIBs with same name in yaml and php ?

**Comment by PipoCanaja** [comment]
"rutos" should probably be renamed "rutos-9xx" and this PR should create "rutos-2xx". Subfolder for both sets of mibs (teltonika/rutos-9xx and teltonika/rutos-2xx).

**Comment by PipoCanaja** [comment]
@jozefrebjak You still have some issues found by travis on tests. Let us know if you need help to solve them.

**Comment by jozefrebjak** [comment]
@PipoCanaja i ran test twice, but it still falling, isn't it because of custom graph ?

**Comment by PipoCanaja** [comment]
> @PipoCanaja i ran tests twice, but it still falling, isn't it because of custom graph ?

According to Travis : 
``` OS rutos-2xx: Discovered sensors data does not match that found in tests/data/rutos-2xx.json ``` 

You need to re-run ./scripts/save-test-data.php for the rutos test files.

**Comment by murrant** [comment]
@jozefrebjak it is because you already have the states in your db, but the code that checks if they need to be updated is case insensitive.

**Comment by jozefrebjak** [comment]
@murrant thank you for help with this.

**Pull request merged**